.github/workflows/setup-gcloud-it.yml

```diff
@@ -4,7 +4,13 @@ on:
   push:
     branches:
     - 'master'
-    pull_request:
```

Comment:
I'm not sure what our desired intended behavior was here. I don't think nesting `pull_request` under `push` is valid. I've updated it here to run on all PRs, but I think that will fail since outside contributors won't have access to the secrets. LMK what our desired behavior is here.

Reply:
Yeah this looks like a typo. For external contributions via forks the `github.repository == github.event.pull_request.head.repo.full_name` would be false and should skip those jobs.